pkg/sriov/sriov.go

```diff
@@ -240,6 +240,13 @@ func (s *sriovManager) ApplyVFConfig(conf *types.NetConf) error {
 		return err
 	}
 
+	// Get vf name after the rebind
+	linkName, err := utils.GetVFLinkNames(conf.DeviceID)
+	if err != nil || linkName == "" {
+		return fmt.Errorf("failed to get VF %s name after rebind with error, %q", conf.DeviceID, err)
+	}
+	conf.HostIFNames = linkName
```

Comment:
as discussed we want HostIFNames to contain the original link name for best effort restore during cmdDel.

during SetupVf(), need to recalculate the link name (add a note explaining there why),  or you could change method signature and pass in the link name instead of taking it from conf.HostIFNames

Reply:
added